.github/workflows/importer.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow automates daily imports of new releases for legacy packages.
+name: Import Packages
+
+# The importer is run every day at 07:00 UTC and can also be run manually from
+# the 'Actions' tab.
+on:
+  # TODO: DISABLE
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 7 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.PACCHETTIBOTTI_TOKEN }}
+      PACCHETTIBOTTI_TOKEN: ${{ secrets.PACCHETTIBOTTI_TOKEN }}
+      SPACES_KEY: ${{ secrets.SPACES_KEY }}
+      SPACES_SECRET: ${{ secrets.SPACES_SECRET }}
+
+    steps:
+      # Checks out the registry-dev repository so we can run scripts from it
+      - uses: actions/checkout@v3
+        with:
+          repository: purescript/registry
+          # TODO: DISABLE
+          ref: trh/single-vs-batch
```

Comment:
We need to remove this line once the `trh/single-vs-batch` branch merges in the `registry` repo.